examples/drivers/m5stickc-pedometer/main.js

```diff
@@ -18,15 +18,16 @@ import Resource from "Resource";
 import loadJPEG from "commodetto/loadJPEG";
 import Timer from "timer";
 import ILI9341 from "ili9341";
+import config from "mc/config";
 
 const STARTING_STEPS = 123;
 const GOAL = 200;
 const SHAKEVALUE = 3;
 
-let pixelsOut = new ILI9341({});
-const width = pixelsOut.width;
-const height = pixelsOut.height;
-const render = new Poco(pixelsOut, {rotation: 90});
+const rotation = config.rotation;
+const width = (rotation / 90) % 2 ? screen.height: screen.width;
+const height =(rotation / 90) % 2 ? screen.width: screen.height;
+let render = new Poco(screen, {width, height, rotation});
```

Comment:
Poco does not use the `width` and `height` from the options object. It [takes](https://github.com/Moddable-OpenSource/moddable/blob/968fe02408be0343a64c2b627266cb85af316267/modules/commodetto/commodettoPoco.js#L44-L45) `width` and `height` directly from the `screen` instance and then [applies](https://github.com/Moddable-OpenSource/moddable/blob/968fe02408be0343a64c2b627266cb85af316267/modules/commodetto/commodettoPoco.c#L77) the rotation. So, I think this can be simplified. Would you trying something like this?

```js
 let render = new Poco(screen, {rotation: config.rotation});
 const width = render.width, height = render.height;
```